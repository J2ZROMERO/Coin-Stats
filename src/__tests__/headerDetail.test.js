import React from 'react';
import { render } from '@testing-library/react';
// eslint-disable-next-line no-unused-vars
import headerDetails from '../components/headerDetail';

describe('Unit test', () => {
  it('Should render details of coins and compare with Snapshot', () => {
    const { asFragment } = render(<headerDetails name="some" />);
    expect(asFragment(<headerDetails name="some" />)).toMatchSnapshot();
  });
});
