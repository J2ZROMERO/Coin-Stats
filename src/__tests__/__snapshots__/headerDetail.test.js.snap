// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unit test Should render details of coins and compare with Snapshot 1`] = `
<DocumentFragment>
  <headerdetails
    name="some"
  />
</DocumentFragment>
`;
